test/metabase/api/embed_test.clj

```diff
@@ -1949,98 +1695,161 @@
-      (is (= {action_eid               {:id action_id               :type :action            :status :ok}
-              collection_eid           {:id collection_id           :type :collection        :status :ok}
-              core_user_eid            {:id core_user_id            :type :user              :status :ok}
-              dashboard_tab_eid        {:id dashboard_tab_id        :type :dashboard-tab     :status :ok}
-              dimension_eid            {:id dimension_id            :type :dimension         :status :ok}
-              native_query_snippet_eid {:id native_query_snippet_id :type :snippet           :status :ok}
-              permissions_group_eid    {:id permissions_group_id    :type :permissions-group :status :ok}
-              pulse_eid                {:id pulse_id                :type :pulse             :status :ok}
-              pulse_card_eid           {:id pulse_card_id           :type :pulse-card        :status :ok}
-              pulse_channel_eid        {:id pulse_channel_id        :type :pulse-channel     :status :ok}
-              card_eid                 {:id card_id                 :type :card              :status :ok}
-              dashboard_eid            {:id dashboard_id            :type :dashboard         :status :ok}
-              dashboardcard_eid        {:id dashboardcard_id        :type :dashboard-card    :status :ok}
-              segment_eid              {:id segment_id              :type :segment           :status :ok}
-              timeline_eid             {:id timeline_id             :type :timeline          :status :ok}}
-             (api.embed.common/model->entity-ids->ids
-              {:action            [action_eid]
-               :card              [card_eid]
-               :collection        [collection_eid]
-               :dashboard         [dashboard_eid]
-               :dashboard-card    [dashboardcard_eid]
-               :dashboard-tab     [dashboard_tab_eid]
-               :dimension         [dimension_eid]
-               :permissions-group [permissions_group_eid]
-               :pulse             [pulse_eid]
-               :pulse-card        [pulse_card_eid]
-               :pulse-channel     [pulse_channel_eid]
-               :segment           [segment_eid]
-               :snippet           [native_query_snippet_eid]
-               :timeline          [timeline_eid]
-               :user              [core_user_eid]}))))))
-
-(deftest missing-entity-translations-test
-  (is (= {"abcdefghijklmnopqrstu" {:type :card, :status :not-found}}
-         (api.embed.common/model->entity-ids->ids {:card ["abcdefghijklmnopqrstu"]}))))
-
-(deftest wrong-format-entity-translations-test
-  (is (= {"abcdefghijklmnopqrst"
-          {:type :card,
-           :status :invalid-format,
-           :reason ["\"abcdefghijklmnopqrst\" should be 21 characters long, but it is 20"]}}
-         (api.embed.common/model->entity-ids->ids {:card ["abcdefghijklmnopqrst"]}))))
```

Comment:
I'm not familiar with these `model->entity-ids->ids` tests. Why are they removed here?